scripts/google_examples.py

```diff
@@ -33,19 +34,24 @@ def runTests():
         "22 --pageSize 1",
     ]
     for command in commands:
-        cmdStr = "python {path} {keyStr} {workarounds} {command} {baseUrl}"
+        cmdStr = """
+            python {path} {keyStr} {minimalOutput} {workarounds}
+            {command} {baseUrl}
+            """
         cmdDict = {
             "path": path,
             "command": command,
             "workarounds": workarounds,
             "keyStr": keyStr,
             "baseUrl": baseUrl,
+            "minimalOutput": minimalOutput,
         }
         cmd = cmdStr.format(**cmdDict)
+        splits = cmd.split()
+        cleanCmd = ' '.join(splits)
         print(separator)
-        print(cmd)
+        print(cleanCmd)
         print(separator)
-        splits = cmd.split()
         proc = subprocess.Popen(splits)
```

Comment:
Yes, but that way suppresses all the output of the command, which we don't want.